mautic/mautic
# Fix report date filter

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The dates in report filters were convert to UTC format.  This make issue with filtering. 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create custom field: **Test date**
3.  Create contact and fill **Test date** to 26. 1. 2022
4. Make sure your timezone in Configuration is **Europe/Paris** for example
5. Create contacts report with filter  **Test date** equal 26. 1. 2022
6. Before patch your contact is not in report
7. After patch contact go to report properly




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10800"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
